src/librustc/middle/lang_items.rs

```diff
@@ -90,6 +90,10 @@ impl LanguageItems {
         }
     }
 
+    pub fn require_owned_box(&self) -> Result<ast::DefId, String> {
```

Comment:
> the 'right' thing should be at least as ergonomic as the 'wrong' thing (.owned_box())

Hm, actually, I'm not totally clear why `owned_box` is wrong? Doesn't it return an `Option`, and the ICE caused by the `unwrap` call?